README.md

```diff
@@ -3,7 +3,10 @@
 [![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.me/TheAlgorithms/100) &nbsp;
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/TheAlgorithms) &nbsp;
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/TheAlgorithms/Python)
-
+[![HitCount](http://hits.dwyl.com/QuantumNovice/TheAlgorithms/Python.svg?style=flat)](http://hits.dwyl.com/TheAlgorithms/Python)
+[![Inline docs](http://inch-ci.org/github/TheAlgorithms/Python?branch=master)](http://inch-ci.org/github/TheAlgorithms/Python)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/TheAlgorithms/Python/issues)
+[![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/downloads/release/python-370/)
```

Comment:
Let's make this __Tested on Python 3.7__ because I do not think that we require our users to be on 3.7 vs. 3.6.

Reply:
Please fix the title of this PR to something more descriptive like: __Add badges to the top of README.md__

Should we add a badge that tells the reader if our Travis CI tests are passing?

Reply:
I added badge for Travis-CI.
Also should we be using Circle CI along with travis-CI?